apps/server/src/modules/files-storage/uc/files-storage.uc.ts

```diff
@@ -281,12 +278,12 @@ export class FilesStorageUC {
 			),
 		]);
 
-		const [newRecord] = await this.copy(userId, [fileRecord], copyFileParams.target);
+		const [newRecord] = await this.storageCopy(userId, [fileRecord], copyFileParams.target);
 
 		return newRecord[0];
 	}
 
-	private async copy(
+	private async storageCopy(
```

Comment:
Why is this being renamed to `storageCopy`? Apparently it's not only about the storage but involves the `FileRecord` too.